### PR TITLE
Reverse chronological ordering of comments, fixes #723

### LINKED ISSUE #723: Order WI comments reverse-chronologically
Latest comment should be the first one.

Part of userstory: #651 (?)

### PR DESCRIPTION
**Pull request by debloper**
Signed-off-by: Soumya Deb <debloper@gmail.com>

### CODE REVIEW

File: src/app/work-item/work-item-list/work-item-detail/work-item-comment/work-item-comment.component.html [hunk 1]
```diff
@@ -8,24 +8,24 @@
...
-                        <img id="{{'comment_avatar_' + counter}}"
-                        class="user-assign-avatar pull-left" 
+                        <img class="user-assign-avatar pull-left"
                         tooltipPlacement="right"
                         tooltip="{{comment.relationalData.creator?.attributes?.fullName}}"
                         *ngIf="comment.relationalData.creator?.attributes?.imageURL" 
                         [src]="comment.relationalData.creator?.attributes?.imageURL | almAvatarSize:30"
                         />
                     </div>
-                    <small class="pull-right createTimeBlock" id="{{'comment_time_' + counter}}">{{comment.attributes['created-at'] | date:'medium'}}</small>
```
Comment (sanbornsen) [inline src/app/work-item/work-item-list/work-item-detail/work-item-comment/work-item-comment.component.html]: Why did you remove the ID ? This is for functional testing. @naina-verma Please confirm.

File: src/app/work-item/work-item-list/work-item-detail/work-item-comment/work-item-comment.component.html [hunk 2]
```diff
@@ -8,24 +8,24 @@
...
+                    <small class="pull-right createTimeBlock">{{comment.attributes['created-at'] | date:'medium'}}</small>
                     <div class="comment-body">
-                        <h5  class="col-md-12 col-sm-12 col-xs-12 no-padding truncate"><strong  id="{{'comment_username_' + counter}}">{{comment.relationalData?.creator?.attributes?.fullName}}</strong>
+                        <h5 class="col-md-12 col-sm-12 col-xs-12 no-padding truncate">
+                            <strong>{{comment.relationalData?.creator?.attributes?.fullName}}</strong>
                         </h5>
                     </div>
                 </div>
-                <p class="col-md-12 col-xs-12 col-sm-12" id="{{'comment_body_' + counter}}">{{comment.attributes.body}}</p>
```
Comment (sanbornsen) [inline src/app/work-item/work-item-list/work-item-detail/work-item-comment/work-item-comment.component.html]: Why did you remove the ID ? This is for functional testing. @naina-verma Please confirm.

File: src/app/work-item/work-item-list/work-item-detail/work-item-comment/work-item-comment.component.html (hunk 1 above)
Comment (debloper) [inline src/app/work-item/work-item-list/work-item-detail/work-item-comment/work-item-comment.component.html]: They aren't being used as of now, but reverting just in case they might be.

File: src/app/work-item/work-item-list/work-item-detail/work-item-comment/work-item-comment.component.html (hunk 2 above)
Comment (debloper) [inline src/app/work-item/work-item-list/work-item-detail/work-item-comment/work-item-comment.component.html]: They aren't being used as of now, but reverting just in case they might be.

**Comment by nimishamukherjee** [comment]
[test]
